lib/thor/actions/file_manipulation.rb

```diff
@@ -340,7 +340,8 @@ def capture(*args)
       with_output_buffer { yield(*args) }
     end
 
-    def with_output_buffer(buf = "") #:nodoc:
+    def with_output_buffer(buf = "".dup) #:nodoc:
+      raise "No: #{buf}" if buf.frozen?
```

Comment:
Thanks @rafaelfranca - that was actually a debugging line that I shouldn't have committed in the first place. Have kept the raise, using ArgumentError, and put in a more useful error message.